gammapy/time/models.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+import numpy as np
+from ..utils.modeling import Parameter, ParameterList
+
+__all__ = [
+    'PhaseCurve',
+]
+
+log = logging.getLogger(__name__)
+
+
+class PhaseCurve(object):
```

Comment:
@labsaha - all of the code in the class (including the docstring) should be indented.